Assets/Tests/InputSystem/CoreTests_Editor.cs

```diff
@@ -2593,6 +2594,96 @@ public void Editor_CanRestartEditorThroughReflection()
         EditorHelpers.RestartEditorAndRecompileScripts(dryRun: true);
     }
 
+    [Test]
+    public void DocsContainNoMissingOrUnreferencedImages()
+    {
+        const string docsPath = "Packages/com.unity.inputsystem/Documentation~/";
+        const string imagesPath = "Packages/com.unity.inputsystem/Documentation~/images/";
+        var regex = new Regex("\\(.*images\\/(?<filename>[^\\)]*)", RegexOptions.IgnoreCase);
+
+        // Add files here if you want to ignore them being unreferenced.
+        string[] unreferencedIgnoreList = new [] { "InputArchitectureLowLevel.sdxml" };
```

Comment:
I have no idea what this file is but assume its important ;)

Reply:
It's the source file for the architecture diagram. In a rather obscure format, though (SimpleDiagrams).